packages/components/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
 - `Checkbox`, `Textarea`: `formDisabledCallback` and `formStateRestoreCallback` from ElementInternals API and sync
   validity with form element. ([#3528](https://github.com/porsche-design-system/porsche-design-system/pull/3528))
 
+#### Changed
```

Comment:
- Add to `Added` section that the form prop to associate the elements with a form was added
- Add Note that native validation won't work anymore